packages/foundry/src/bin.ts

```diff
@@ -1,22 +1,46 @@
-import * as process from "process";
+#!/usr/bin/env node
+import { resolve } from "node:path";
+import process from "node:process";
+import { Command } from "commander";
+import { type OptionValues } from "@commander-js/extra-typings";
```

Comment:
Really not convinced of this package `@commander-js/extra-typing`

I understand that it forces options to be typed unknown instead of any, but for a small program like this it feels overkill and we end up casting the values without checking anyway. Might as well just use any

Reply:
And for the create Method, i would just type it 

```
export async function create(templateId: TemplateId, outputDir: string, args: Record<string, string>) {


```